tests/unit/core/TestInstallAddon.js

```diff
@@ -297,6 +297,20 @@ describe(__filename, () => {
         .toEqual('https://a.m.o/files/linux.xpi');
     });
 
+    it('finds a Linux Ubuntu install URL', () => {
```

Comment:
Can we have an extra one that is 100% not Linux but UNIX instead, just for completeness/clarity? I know a user was asking if we could load Linux XPIs for BSD users, so it'd be nice to have a test that spells out that intention and makes sure it stays true.

Reply:
Good call. I had actually made a typo (UNIX not Unix) so it wasn't even working. I'll double check the mapping but it's probably overkill to add a test for each one.

Reply:
Yeah every one is definitely excessive, I was just thinking at least one "Unix". 😄